sccm/apps/deploy-use/import-export-applications.md

```diff
@@ -0,0 +1,46 @@
+---
+
+# Mandatory fields.
+title: Import/Export Applications
+titleSuffix: Configuration Manager
+description: Import and export applications.
+author: chriskibble
+ms.author: NO VALUE SET # Microsoft employees only
+ms.date: 5/6/2019
+ms.topic: conceptual
+ms.technology: configmgr-app
+---
+# Create applications in Configuration Manager
+
+*Applies to: System Center Configuration Manager (Current Branch)*
+
+Configuration Manager allows for the import and export of applications within the console.  This allows you to copy applications in to other environments, such as copying from a test environment into a production environment.
```

Comment:
A suggested rewording:

```suggestion
Use Configuration Manager to import and export applications to another hierarchy. For example, copy an application from a test environment to a production environment.
```

Reply:
Made some minor changes to your suggestion and submitted as #1585